src/utils/rfq_blockchain_utils.ts

```diff
@@ -231,4 +256,56 @@ export class RfqBlockchainUtils {
     public async isWorkerReadyAsync(workerAddress: string, balance: BigNumber, gasPrice: BigNumber): Promise<boolean> {
         return isWorkerReadyAndAbleAsync(this._web3Wrapper, workerAddress, balance, gasPrice);
     }
+
+    /**
+     * Converts a @0x/web3-wrapper `TxData` object into an Ethers `TransactionRequest`.
+     *
+     * If the `from` address is not present in the `TxData`, the function uses the address
+     * of the Ethers Wallet passed to `rfqBlockchainUtils`
+     *
+     * If the `to` address is not present, the function uses the exchange proxy address.
+     */
+    public async transformTxDataToTransactionRequestAsync(
+        txOptions: Partial<TxData>,
+        callData?: utils.BytesLike,
+    ): Promise<providers.TransactionRequest> {
+        let from = txOptions.from;
+        if (!from) {
+            if (!this._ethersWallet) {
+                throw new Error("An Ethers wallet must be provided if no 'from' address is specified");
+            }
+            from = await this._ethersWallet.getAddress();
+        }
+
+        return {
+            to: txOptions.to || this._exchangeProxy.address,
+            from,
+            nonce: txOptions.nonce,
+            gasLimit: EthersBigNumber.from(txOptions.gas),
+            gasPrice: EthersBigNumber.from(txOptions.gasPrice),
+            data: callData,
+            value: EthersBigNumber.from(txOptions.value),
+            // TODO (rhinodavid): Add Chain ID
+        };
+    }
+
+    /**
+     * Uses the Ethers Wallet to sign a transaction. Returns both the signed transaction and its hash.
+     *
+     * If the containing class has been initialized without a wallet, the function throws.
+     */
+    public async signTransactionAsync(
+        transaction: providers.TransactionRequest,
+    ): Promise<{ signedTransaction: string; hash: string }> {
```

Comment:
nit: can we return `transactionHash` instead of the term `hash` - will keep it consistent whenever we use the term `transactionHash`

Reply:
sgtm I was going back and forth about this